### PR TITLE
Fix leftover blank space at bottom of scrollView on android

### PR DESCRIPTION
**Pull request by mikelovesrobots**
Recently we discovered a layout bug on Android where the temporary `extraScrollHeight` bottom-padding never went away after the keyboard was closed.

I dove into the code and noticed that when `keyboardDidHide` (Android) is triggered, `_resetKeyboardSpace` intends to reset the keyboardSpace (bottomPadding) state, but is including `extraScrollHeight` in the final calculation. That doesn't seem right, and removing it fixes our leftover padding issue after the keyboard is closed.

This change doesn't seem to cause any problems on iOS.

### CODE REVIEW

File: lib/KeyboardAwareHOC.js [hunk 1]
```diff
@@ -362,8 +362,8 @@ function listenToKeyboardEvents(ScrollableComponent: React$Component) {
 
     _resetKeyboardSpace = () => {
       const keyboardSpace: number = this.props.viewIsInsideTabBar
-        ? _KAM_DEFAULT_TAB_BAR_HEIGHT + this.props.extraScrollHeight || 0
```
Comment (alvaromb) [inline lib/KeyboardAwareHOC.js]: Have you tried to set `extraScrollHeight` to `0`?

Comment (alvaromb) [reply lib/KeyboardAwareHOC.js]: In fact, if you don't set `extraScrollHeight`, it should have `0` value. Can you check in your code the execution of this ternary operator?

Comment (mikelovesrobots) [reply lib/KeyboardAwareHOC.js]: I'm feeling a bit slow here. Yes. If I set `extraScrollHeight` to 0, then the keyboard space is reset to the appropriate height. But the bug is about when `extraScrollHeight` isn't 0. It's a useful property and it does the right thing when the keyboard is open. When the keyboard is closed, the extra padding it provides should go away and that's what this PR is about.

**Comment by todorone** [comment]
This bug affects iOS also - #289 
@alvaromb Any comments on how this PR can be merged?

**Comment by alvaromb** [comment]
@todorone does this PR solves #289?

**Comment by EQuimper** [comment]
Just test this pr and look like that fix the issue 
aug-24-2018 12-40-18

**Comment by clfristoe** [comment]
This would be a really useful fix! Any timeline on when it will be merged?

**Comment by todorone** [comment]
@alvaromb Is there any help needed to merge this PR?

**Comment by alvaromb** [comment]
https://github.com/APSL/react-native-keyboard-aware-scroll-view/releases/tag/v0.7.4

**Pull request merged**